enterprise/wan-replication/simulating-max-idle/src/main/java/SingleMaxIdleSimulator.java

```diff
@@ -17,22 +17,31 @@ public static void main(String[] args) {
         simulator.simulate();
     }
 
+    // You may want to consider using IdentifiedDataSerializable instead
+    // of default java serialization for better performance. If you do so,
+    // EntryProcessor, key and value should extend IdentifiedDataSerializable.
```

Comment:
I see why you suggest IDS format for the EP, but why should key & value also be IDS format?

Reply:
It was a leftover from a changed backup processor logic, which is reverted now. https://github.com/hazelcast/hazelcast-code-samples/pull/562/commits/c7a40cf81feef1f54f3d435378fe7086f548fc55